tests/integration/kubernetes/confidential/confidential_common.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bats
+# Copyright 2022-2023 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+source "../../tests_common.bash"
+
+setup_unencrypted_confidential_pod() {
+    local pod_name="unencrypted-confidential"
+    local IMAGE_REPO="ghcr.io/confidential-containers/test-container"
+    local UNENCRYPTED_IMAGE_URL="${IMAGE_REPO}:unencrypted"
+    local TEST_DIR="$(mktemp -d /tmp/test-kata-confidential.XXXXXXXX)"
+    export SSH_KEY_FILE="${TEST_DIR}/container-ssh-key"
+
+   	k8s_generate_service_yaml "${pod_config_dir}/${pod_name}.yaml" "${UNENCRYPTED_IMAGE_URL}"
+
+    if [ "${KATA_HYPERVISOR}" == "qemu-sev" ]; then
+    	k8s_yaml_set_annotation "${pod_config_dir}/${pod_name}.yaml" "io.katacontainers.config.guest_pre_attestation.enabled" "false"
```

Comment:
Don't think this annotation is supported in main yet. Fortunately there is no pre-attestation support rn, so we don't need to worry about disabling it.